docs/running_locally.md

```diff
@@ -83,14 +83,17 @@ AUTH_TOKEN=some_random_token
 
 `AUTH_TOKEN` and `CGW_FLUSH_TOKEN` must be the same.
 
-For the Transactions service, follow these steps:
+For the Events service, follow these steps:
```

Comment:
For the future, we should make this automatic, adding a management endpoint on the events service. But it's also good that users know what's going on

Reply:
Agree, at least for the client-gateway.